frontend/device/android/device.lua

```diff
@@ -376,6 +376,9 @@ function Device:showLightDialog()
         if android.isWarmthDevice() then
```

Comment:
just for warmth devices? Don't you need to broadcast frontlight state change after a intensity change?.

Reply:
Ups. Of course!

Reply:
moved at the end of the block